spark/v3.4/spark/src/test/java/org/apache/iceberg/spark/sql/TestPartitionedWrites.java

```diff
@@ -18,12 +18,4 @@
  */
 package org.apache.iceberg.spark.sql;
 
-import java.util.Map;
-
-public class TestPartitionedWrites extends PartitionedWritesTestBase {
-
-  public TestPartitionedWrites(
-      String catalogName, String implementation, Map<String, String> config) {
-    super(catalogName, implementation, config);
-  }
-}
+public class TestPartitionedWrites extends PartitionedWritesTestBase {}
```

Comment:
Can we remove this test? I believe this test will be used as the super-class for other related tests, but this is not actually used now. For now, I keep this test for further use.

Reply:
what do you mean this test isn't used? You can directly execute it and it will also be part of CI runs

Reply:
Ah I was completely wrong, yes this is required. Thanks for pointing that.